frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -66,7 +66,7 @@ function CriticalQuestions(props) {
       dispatch(fetchStory());
     } else {
       setFormValues({ ...formValues, ...story });
-      setRecentTravels(story.travels);
+      if (story.travels) setRecentTravels(story.travels);
```

Comment:
This will also set them if the travels are `[]` (an empty array is truthy). I suggest doing a `.length` check to avoid unnecessary state updates!

Reply:
But an empty array is a valid state for travels, so we should not prevent that from happening
Nowadays at the beginning of a story, `story.travels` is undefined, and stays that way until we submit the story and store it in the sate.
Also, in the future if we add the feature of removing travels, we may need to set the travels as an empty array and that should trigger a state update

Reply:
> Nowadays at the beginning of a story, story.travels is undefined
Ah, that's the bit of information I was missing! This is usually undesirable (it should be an empty array since startup), but let's leave it as it is since it's pretty pressing to merge this.

If we ever do remove travels, that should be handled separately from this - this is the effect run on component startup, not the change handler for travels.